README.md

```diff
@@ -40,6 +40,7 @@ OPENID_CONNECT_VIEWSET_CONFIG = {
     "USE_AUTH_BACKEND": False,
     "AUTH_BACKEND": "",  # Defaults to django.contrib.auth.backends.ModelBackend
     "USE_RAPIDPRO_VIEWSET": False,
+    "FIELD_VALIDATION_REGEX": {"username": "(?!^\d+$)^.+$"} # This is the default validation
```

Comment:
Can we have a comment describing the validation?

Reply:
Changed in the latest commit

Reply:
@DavisRayM I suggest we have a descriptive message when the validation failed. Something like
```
`"FIELD_VALIDATION_REGEX": [{
          "Field": "username",
          "regex" : "(?!^\d+$)^.+$",
          "message": "Username should only contain alpha numeric characters"
}]
```
I feel this message `raise ValueError(f'Invalid `{k}` value `{data[k]}`')` is too ambiguous. We need to be raising more descriptive error messages.